docs/plugin/README.md

```diff
@@ -8,8 +8,9 @@ The plugin main file is `plugin.js`
 
 ## How to install one plugin?
 
-1. Find it on we.js site on npm with `npm searh we-plugin-` or in github search
-2. Install it with `npm install [pluginName]` like `npm install we-plugin-form`
+1. Find on we.js site on npm with `npm search we-plugin-` or in github search
```

Comment:
Find plugins in we.js site, npm with command `npm search we-plugin-` or in github search